src/pipecat/services/cartesia.py

```diff
@@ -210,7 +210,6 @@ def _get_websocket(self):
     async def _handle_interruption(self, frame: StartInterruptionFrame, direction: FrameDirection):
         await super()._handle_interruption(frame, direction)
         await self.stop_all_metrics()
-        await self.push_frame(LLMFullResponseEndFrame())
```

Comment:
I already mentioned it in the commit messages but this is not needed. The context aggregator interruption already handle interruption properly. We don't want to pollute the code with `push_frame()`  otherwise it will become very hard to maintain.